### PR TITLE
Fix BG popping

### PR DESCRIPTION
**Pull request by charasyn**
The BG popping issue was fixed by ensuring the background layers were all lined up and filter was disabled (I think it was already on most of these, but I think I just didn't reimport them before).

I'd recommend deleting your .import folder if you try this out locally (and once it's merged into main) to force Godot to reimport all the files. I was also getting warnings after deleting some unused imports (old images that were removed) so I removed them from the tilesets.

At the moment, this is based on the "settings fix" branch since it requires the project settings to be correct to look 100% nice - that should be merged into main first, then this can be rebased on main.

### CODE REVIEW

File: Documentation/save_info.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Save file information
+To save high score information, the scores are persisted to a .JSON file (see `godotproject\Menus\HighScoreScreen\HighScoreScreen.gd` for more information).
+
+This file is saved as `user://scores.json`, and a custom user directory of `fgts-data` is set in the Godot project settings.
+
+On the Pi, the path will be
+`/home/pi/.local/share/fgts-data/scores.json`
+
+On Windows, the path will be
+`%APPDATA%\fgts-data\scores.json`
```
Comment (charasyn) [inline Documentation/save_info.md:10]: This file is actually part of #88, which this is based on

File: godotproject/Bird.gd [hunk 2]
```diff
@@ -83,7 +83,7 @@ func take_damage(damage):
 		$Orientation/Sprite.play("hurt")
 		# Handle enemy health
 		health = health - damage
-		attackState == STATE.idle_e
+		attackState = STATE.idle_e
```
Comment (charasyn) [inline godotproject/Bird.gd:86]: Fixes a warning we were getting in the second level

File: godotproject/Level0.tscn [hunk 3]
```diff
@@ -35,6 +35,7 @@ margin = 10.0
 player_nodepath = NodePath("../Player")
 
 [node name="DeathPlane" parent="." instance=ExtResource( 4 )]
+visible = false
```
Comment (charasyn) [inline godotproject/Level0.tscn:38]:
```suggestion
```

File: godotproject/Level2.tscn [hunk 4]
```diff
@@ -1066,8 +958,8 @@ scale = Vector2( 0.5, 1 )
 
 [node name="BlinkingComputer" parent="." instance=ExtResource( 24 )]
 position = Vector2( 593.788, 815.941 )
-frame = 20
+frame = 15
```
Comment (charasyn) [inline godotproject/Level2.tscn:961]:
```suggestion
frame = 20
```

File: godotproject/Level2.tscn [hunk 5]
```diff
@@ -1066,8 +958,8 @@ scale = Vector2( 0.5, 1 )
...
 
 [node name="BlinkingComputer2" parent="." instance=ExtResource( 24 )]
 position = Vector2( 3080, 736 )
-frame = 20
+frame = 15
```
Comment (charasyn) [inline godotproject/Level2.tscn:965]:
```suggestion
frame = 20
```

File: godotproject/PlayerTongue.gd [hunk 6]
```diff
@@ -50,7 +50,7 @@ func start_idle():
 	$ShootSoundPlayer.playing = false
 	$StickSoundPlayer.playing = false
 	
-	$Area2D/CollisionShape2D.disabled = true
+	$Area2D/CollisionShape2D.set_deferred("disabled", true)
```
Comment (charasyn) [inline godotproject/PlayerTongue.gd:54]: Fixes a warning

**Comment by fheysel** [comment]
Changes look good, Im a little confused about how we get this working on our side. Will pulling main not be sufficient?

**Comment by charasyn** [comment]
It may not be, you should:
1. close editor
2. pull main
3. delete godotproject/.import folder
4. open editor

It should be fine after that

**Comment by fheysel** [comment]
ok thanks, make sure this is communicated to the rest of the team

**Pull request merged**